recipes/python-toolchain/meta.yaml

```diff
@@ -0,0 +1,37 @@
+package:
+  name: python-toolchain
+  version: 1.0.0
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python
+
+  run:
+    - python
+    - setuptools
+    - pip
+
+test:
+  commands:
+    - test -f "${PREFIX}/etc/pip.conf"                                 # [unix]
+   #- test -f "${STDLIB_DIR}/distutils/distutils.cfg"                  # [unix]
+    - if not exist "%LIBRARY_PREFIX%\\etc\\pip.conf" exit 1            # [win]
+   #- if not exist "%STDLIB_DIR%\\distutils\\distutils.cfg" exit 1     # [win]
```

Comment:
Oh, and another problem: conda build does not activate the environment but only changes the PATH: at least the gopath isn't set in during tests :-(

```
The following NEW packages will be INSTALLED:

    go: 1.4.2-0

Fetching packages ...
go-1.4.2-0.tar 100% |###############################| Time: 0:00:00 130.22 MB/s
Extracting packages ...
[      COMPLETE      ]|##################################################| 100%
Linking packages ...
[      COMPLETE      ]|##################################################| 100%

C:\portabel\miniconda\conda-bld\test-tmp_dir>go version
go: cannot find GOROOT directory: C:\portabel\miniconda\envs\_build\go

C:\portabel\miniconda\conda-bld\test-tmp_dir>if errorlevel 1 exit 1
TESTS FAILED: go-1.4.2-0
```

I'm currently thinking about this workaround:

```
  commands:
    - activate _test  # [win]
    - go version
    - go help
    - go run hello.go
    - cmd /c echo x%GOROOT% NEQ x%CONDA_DEFAULT_ENV%\go  # [win]
    - cmd /c if x%GOROOT% NEQ x%CONDA_DEFAULT_ENV%\go exit 1   # [win]
```